.github/CODEOWNERS

```diff
@@ -5,4 +5,4 @@
 # These are the default owners for the code and will
 # be requested for review when someone opens a pull request.
 
-* @abhi16394 @bharathsreekanth @JacobGros @yifeijin @mjsdell @jooseppi-luna
+* @abhi16394 @bharathsreekanth @JacobGros @yifeijin @mjsdell @jooseppi-luna @ashleyvjoy @Deepak-Ghivari @khareRajshree @KshitijaKakde @prabhu-dell @VamsiSiddu-7
```

Comment:
could you remove @yifeijin from this list. thanks

Reply:
done.